QualityControl/test/api/layouts/api-patch-layout.test.js

```
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { suite, test } from 'node:test';
+import { GLOBAL_TEST_TOKEN, OWNER_TEST_TOKEN, URL_ADDRESS } from '../config.js';
+import request from 'supertest';
+
+export const apiPatchLayoutTests = () => {
+  suite('PATCH /layout/:id', () => {
+    test('should return a 404 error if the id of the layout does not exist', async () => {
+      await request(`${URL_ADDRESS}/api/layout/`)
+        .patch(`?token=${OWNER_TEST_TOKEN}`)
+        .expect(404, {
+          error: '404 - Page not found',
+          message: 'The requested URL was not found on this server.',
+        });
+    });
+
+    test('should return a 404 error if the id is not provided', async () => {
```

I believe you miss-copied the description of the test.
You are saying in this test `should return a 404 error if the id is not provided` but then in the test, the id is provided `test` 

Then in the test above you are saying `should return a 404 error if the id of the layout does not exist` but you are not providing any id to check

Probably you swapped by mistake between the two

---

Sure, it was a mistake. Done.